docs/reference/docs/update.asciidoc

```diff
@@ -139,13 +162,19 @@ the request was ignored.
 [source,js]
 --------------------------------------------------
 {
+   "_shards": {
+        "total": 0,
+        "successful": 0,
+        "failed": 0
+   },
    "_index": "test",
    "_type": "type1",
    "_id": "1",
-   "_version": 1,
+   "_version": 6,
    "result": noop
```

Comment:
I see the trouble. It might make more sense to move this `// TESTRESPONSE` snippet above the example of disabling `detect_noop`.